src/index.js

```diff
@@ -0,0 +1,234 @@
+let name1 = "Abraham";
+let count1 = 0;
+for (let i = 0; i < name1.length; i++) {
+  // Your code here
+  if (name1[i].toLowerCase() === "a") {
+    count1++;
+  }
+}
+console.log("[pregunta #1]: ", count1); // Answer should be: 3
+
+// Question 2: Reverse the string "Moses".
+let name2 = "Moses";
+let reversedName2 = "";
+for (let j = name2.length - 1; j >= 0; j--) {
+  // Your code here
+  reversedName2 += name2[j];
+}
+console.log("[pregunta #2]: ", reversedName2); // Answer should be: sesoM
+
+// Question 3: Sum the integers from 1 to 12 (the number of sons of Jacob).
+let sum3 = 0;
+for (let k = 1; k <= 12; k++) {
+  // Your code here
+  sum3 += k;
+}
+console.log("[pregunta #3]: ", sum3); // Answer should be: 78
+
+// Question 4: Create a string that contains the first 5 books of the Old Testament separated by a comma.
+let books4 = ["Genesis", "Exodus", "Leviticus", "Numbers", "Deuteronomy"];
+let bookString4 = "";
+for (let l = 0; l < books4.length; l++) {
+  // Your code here
+  bookString4 += books4[l];
+  if (l < books4.length - 1) {
+    bookString4 += ", ";
+  }
+}
+console.log("[pregunta #4]: ", bookString4); // Answer should be: Genesis, Exodus, Leviticus, Numbers, Deuteronomy
+
+// Question 5: Count the number of vowels in the name "Jerusalem".
+let name5 = "Jerusalem";
+let count5 = 0;
+for (let m = 0; m < name5.length; m++) {
+  // Your code here
+  let vowels = name5.charAt(m).toLowerCase();
+  if (
+    vowels === "a" ||
+    vowels === "e" ||
+    vowels === "i" ||
+    vowels === "o" ||
+    vowels === "u"
+  ) {
+    count5++;
+  }
+}
+console.log("[pregunta #5]: ", count5); // Answer should be: 4
+
+// Question 6: Find the product of the numbers 1 to 5.
+let product6 = 1;
+for (let n = 1; n <= 5; n++) {
+  // Your code here
+  product6 *= n;
+}
+console.log("[pregunta #6]: ", product6); // Answer should be: 120
+
+// Question 7: Create a string from an array that tells the story of David and Goliath.
+let story7 = ["David", "fought", "Goliath", "and", "won."];
+let storyString7 = "";
+for (let o = 0; o < story7.length; o++) {
+  // Your code here
+  storyString7 += story7[o] + " ";
+}
+console.log("[pregunta #7]: ", storyString7); // Answer should be: David fought Goliath and won.
+
+// Question 8: Create an acronym from the phrase "In The Beginning".
+let phrase8 = "In The Beginning";
+let acronym8 = "";
+for (let p = 0; p < phrase8.length; p++) {
+  // Your code here
+  if (p === 0) {
+    acronym8 += phrase8[p];
+  }
+  if (phrase8[p] === " ") {
+    acronym8 += phrase8[p + 1];
+  }
+}
+console.log("[pregunta #8]: ", acronym8.toUpperCase()); // Answer should be: ITB
+
+// Question 9: Convert the string "Noah" to uppercase.
+let name9 = "Noah";
+let upperName9 = "";
+for (let q = 0; q < name9.length; q++) {
+  // Your code here
+  upperName9 = name9.toUpperCase();
+}
+console.log("[pregunta #9]: ", upperName9); // Answer should be: NOAH
```

Comment:
Esto es mejor hacer el uppercase al final en el console.log() si no, lo estas haciendo en cada vuelta del for loop un total de 4 veces. No crees que es mejor?